src/binding/defaultBindings/event.js

```diff
@@ -21,6 +21,10 @@ ko.bindingHandlers['event'] = {
                 ko.utils.registerEventHandler(element, eventName, function (event) {
                     var handlerReturnValue;
                     var handlerFunction = valueAccessor()[eventName];
+                    if (handlerFunction !== null && handlerFunction !== undefined && typeof handlerFunction !== "function") {
```

Comment:
you could just do `typeof handlerFunction !== function`, the null and undefined check seems redundant in my opinion, shouldn't those be covered by the `!== function` check?

Reply:
No, in this case this stricter condition was intentional. I wanted to *allow* `null` or `undefined` values as those can be kind of valid values in some cases.

Reply:
Ah, good point. I was under the assumption we didn't want `null` or `undefined` here. Thanks for the explanation.